package.json

```diff
@@ -45,8 +45,10 @@
     "@vitalets/google-translate-api": "^9.2.0",
     "bing-translate-api": "^2.9.1",
     "commander": "^13.1.0",
+    "dotenv": "^16.4.7",
```

Comment:
Dotenv is not being used currently, we can remove this dependency.

Dont worry, it would be added on a refactor but as dev dependency only